editor/src/messages/portfolio/document/overlays/utility_types.rs

```diff
@@ -357,7 +357,10 @@ impl OverlayContext {
 
 		// Hover ring
 		if show_hover_ring {
-			let mut fill_color = graphene_std::Color::from_rgb_str(COLOR_OVERLAY_BLUE.strip_prefix('#').unwrap()).unwrap().with_alpha(0.5).rgba_hex();
+			let mut fill_color = graphene_std::Color::from_rgb_str(COLOR_OVERLAY_BLUE.strip_prefix('#').unwrap())
```

Comment:
should this method be renamed to from_rgb_str_srgb?

Reply:
Probably but there are likely dozens that need that kind of renaming and I plan to revisit this code soon. It needs a *lot* more changes since it's extremely loosy-goosy with gamma and premult at the moment.